### DISCUSSION TITLE
请问文章页面的Vendor Settings怎样关闭啊？

### DISCUSSION THREAD

**Discussion by LZHMS** [ppoffice/hexo-theme-icarus#1224] [General]
![image](https://github.com/ppoffice/hexo-theme-icarus/assets/131608290/fcf8df40-4f8d-4f0c-896a-210e3828e95e)
这个我不清楚是在哪个地方配置的，其他博客似乎都没有这个，问一下这个需要在哪里修改

**Comment by ppoffice** [comment]
@LZHMS 这个不是Icarus引入的。你是不是使用了什么其他插件，或者修改了什么布局代码？
